src/mlpack/tests/rl_components_test.cpp

```diff
@@ -57,15 +65,24 @@ BOOST_AUTO_TEST_CASE(SimplePendulumTest)
  */
 BOOST_AUTO_TEST_CASE(SimpleContinuousMountainCarTest)
 {
-  const ContinuousMountainCar task = ContinuousMountainCar();
+  ContinuousMountainCar task = ContinuousMountainCar(-1.2, 0.6, 0.45, -0.07,
```

Comment:
Might be a good idea to introduce a method to set the maximal number of steps? That way we could use the default constructor.